.github/workflows/ci.yml

```diff
@@ -103,7 +103,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        rails_version: [5.2.5, 6.0.3.6, 6.1.3.1, main]
+        rails_version: [5.0.2.7, 5.2.5, 6.0.3.6, 6.1.3.1, main]
```

Comment:
`5.0.7.2` or `5.0.2.7` ?

Reply:
See https://github.com/github/view_component/commit/6528d4cad8a33e920880b4646792fe95358bd5cf
That said, it affects any version of rails before 5.2